src/index.ts

```diff
@@ -301,10 +301,7 @@ export const init = function (
     button.style.display = 'block';
     button.style.marginBottom = '8px';
     container.appendChild(button);
```

Comment:
As per my comment here: https://github.com/jschanker/suggested-blocks-blockly/pull/14#discussion_r2265091906, instead of appending the `button` and `input` directly to the passed container, let's create a `div` for these elements and append the `div` to the container. So, we can remove the block styling for input and button as we want them to appear on the same line (using their default `inline` styling).

Then set the styling of this div to have fixed position and a top/right position of 0 to fix it in the top right corner of the browser window.